src/Dialog.js

```diff
@@ -110,7 +111,7 @@ export default class Dialog extends Component {
     const {center} = this.props
     const width = this.props.default.width
     const x = center && width ? (window.innerWidth / 2) - (width / 2) : this.props.default.x
-    const defaultProps = {...this.props.default, x}
+    const defaultProps = {...this.props.default, x, this.props.zIndex}
```

Comment:
i think you meant this:

```js
const defaultProps = {...this.props.default, x, zIndex: this.props.zIndex}
```

Reply:
actually i'm not even sure this matters since it's only used directly on line 83.

Reply:
I agree I dont think it is needed